download.dd

```diff
@@ -1,9 +1,5 @@
 Ddoc
 
-<!-- Github ribbon -->
-<a href="http://github.com/D-Programming-Language"><img id="github-ribbon"
-src="images/github-ribbon.png" alt="Fork D on GitHub"></a>
-
```

Comment:
I don't think we need a link to GitHub on the downloads page, but here you go. Added a commit: https://github.com/aG0aep6G/dlang.org/commit/2d1d95dd7a475d5d258332f7bb01796e14966186